source/numerics.tex

```diff
@@ -10238,3 +10239,86 @@
 \end{itemdescr}
 
 \indextext{mathematical special functions|)}
+
+\rSec1[numbers]{Numbers}
+
+\rSec2[numbers.syn]{Header \tcode{<numbers>} synopsis}
+\indexhdr{numbers}%
+
+\begin{codeblock}
+namespace std {
+  namespace numbers {
+    template<typename T> inline constexpr T e_v          = @\unspec@;
```

Comment:
Should be `template<class T>` (although a few uses of `typename` have crept in with `bit_cast` and the date library).

Reply:
Fixed. Created #3103 to fix bit_cast.